spec/requests/registrations_spec.rb

```diff
@@ -1,28 +1,46 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/BlockLength
 RSpec.describe 'Registrations', type: :request do
-
-  shared_examples "sucessful password update" do
-    let(:update_params) { { password: new_password, password_confirmation: new_password } }
+  shared_examples 'successfully updates user and redirects' do
     before do
       sign_in user
       put user_registration_path, params: update_params
+      user.reload
     end
 
     it { expect(response).to redirect_to(edit_user_registration_path) }
+    it { expect(user.comment_notify).to be(false) }
```

Comment:
Random attribute to check if attribute's update is ok.

Reply:
I think this makes the test a bit tricky to follow. It doesn't feel explicit to me since now every test would be dependent on `comment_notify` to be false otherwise it'll be a flaky test which fails once that case isn't met. 

I think following [Single expectation test](https://www.betterspecs.org/#single), it's best to just isolate to just one assertion and test that it redirects. 

Maybe for this, we should ditch the 

```ruby
it { expect(user.comment_notify).to be(false) }
```

and rename this to `shared_examples "successfully redirects"`

Reply:
I agree with this